### DISCUSSION TITLE
Breadcrumbs with trailing slash

### DISCUSSION THREAD

**Discussion by synnottd** [twilio-labs/paste#1277] [Q&A] [answered]
Hi, 

I have some designs that look like the following 
<img width="535" alt="Screen Shot 2021-03-11 at 10 10 04 AM" src="https://user-images.githubusercontent.com/3785920/110771073-00f64200-8252-11eb-9e56-14cb471c2426.png">
e.g. the breadcrumbs have a trailing slash - `Create new API key` is the page header

From the docs I was hoping to be able to use the `last` prop on `BreadcrumbItem` but it seems like `Breadcrumb` overrides that https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/breadcrumb/src/index.tsx#L84.

Currently I've just added an empty `BreadcrumbItem` which seems pretty bad. Would it make sense to change it so `last` is respected? And the line above only gives a default?

**Comment by TheSisb** [comment, marked as answer]
Hi @synnottd,

Good question, and one that isn't addressed in our docs! (Don't worry, I made a ticket in our backlog to fix that soon).  I checked with our designers and the recommendation is _not_ to use a trailing slash in the breadcrumb.  We will make that more clear in our documentation and design assets.

Thanks for checking in!
